pkg/thing/handler/amqp/msg_handler.go

```diff
@@ -21,6 +21,33 @@ type MsgHandler struct {
 	thingInteractor interactors.Interactor
 }
 
+// NewMsgHandler constructs the MsgHandler
+func NewMsgHandler(logger logging.Logger, amqp *network.Amqp, interactor interactors.Interactor) *MsgHandler {
```

Comment:
interactor or thingInteractor. Which one do you think is better?

Reply:
I think `thingInteractor` would be better. I'll change it.

Reply:
Done.